Documentation/operations/troubleshooting.rst

```diff
@@ -468,6 +437,31 @@ If pod networking is not managed by Cilium. Ingress and egress policy rules
 selecting the respective pods will not be applied. See the section
 :ref:`network_policy` for more details.
 
+For a quick assessment of whether any pods are not managed by Cilium, the
+`Cilium CLI <https://github.com/cilium/cilium-cli>`_ will print the number
+of managed pods. If this prints that all of the pods are managed by Cilium,
+then there is no problem:
+
+.. code-block:: shell-session
+
```

Comment:
```suggestion
.. code-block:: shell-session

   $ cilium status
```
Not really a shell session otherwise :) But mostly to let users know how they should get this output.

Reply:
:+1: I got mixed up between the boilerplate for the RST in the docs vs. the boilerplate of writing out the commandline and accidentally dropped this line. Fixed.